Tests?

How should tests be run?

I get 3 test failures using `python setup.py test` or `pytest` using Python 3.6.

```
======================================================================
FAIL: test_forest_regressor (tests.test_treeinterpreter.TestTreeinterpreter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/c97346/p/treeinterpreter/tests/test_treeinterpreter.py", line 75, in test_forest_regressor
    self.assertTrue(np.allclose(base_prediction, pred))
AssertionError: False is not true

======================================================================
FAIL: test_forest_regressor_joint (tests.test_treeinterpreter.TestTreeinterpreter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/c97346/p/treeinterpreter/tests/test_treeinterpreter.py", line 90, in test_forest_regressor_joint
    self.assertTrue(np.allclose(base_prediction, pred))
AssertionError: False is not true

======================================================================
FAIL: test_tree_regressor (tests.test_treeinterpreter.TestTreeinterpreter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/c97346/p/treeinterpreter/tests/test_treeinterpreter.py", line 40, in test_tree_regressor
    self.assertTrue(np.allclose(base_prediction, pred))
AssertionError: False is not true
```

--- a/treeinterpreter/treeinterpreter.py
+++ b/treeinterpreter/treeinterpreter.py
@@ -57,6 +57,8 @@ def _predict_tree(model, X, joint_contribution=False):
     if len(values.shape) == 0:
         values = np.array([values])
     if isinstance(model, DecisionTreeRegressor):
+        # we require the values to be the same shape as the biases
+        values = values.squeeze(axis=1)
         biases = np.full(X.shape[0], values[paths[0][0]])
         line_shape = X.shape[1]
     elif isinstance(model, DecisionTreeClassifier):

